paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -655,6 +655,9 @@
                         "OrderedReady",
                         "Parallel"
                     ]
+                },
+                "is_istio_sidecar_injection_enabled": {
+                    "type": "boolean"
```

Comment:
Maybe add this?

`"default": false`

Otherwise lgtm

Reply:
It's taking false as a default value here, https://github.com/Yelp/paasta/blob/master/paasta_tools/kubernetes_tools.py#L1446 , So i think we can skip adding here